source/appModules/explorer.py

```diff
@@ -224,16 +223,6 @@ def _get_value(self):
 			return value
 		return value.replace(CHAR_LTR_MARK,'').replace(CHAR_RTL_MARK,'')
 
-class ReadOnlyEditBox(IAccessible):

```

Comment:
Note that this class is also imported in the `dllhost` appModule.

Reply:
this appears to be an API breaking change as well